### PR TITLE
Replace symbols for assembly references, see #7232

### PR DESCRIPTION
**Pull request by Taronyu**
I added a call to ReplaceSymbols when the references are processed. Hope that was all to fix this.

### CODE REVIEW

File: Tools/MonoGame.Content.Builder/BuildContent.cs [hunk 1]
```diff
@@ -306,7 +306,7 @@ public void Build(out int successCount, out int errorCount)
             // so it can resolve importers, processors, writers, and types.
             foreach (var r in References)
             {
-                var assembly = r;
+                var assembly = ReplaceSymbols(r);
```
Comment (tomspilman) [inline Tools/MonoGame.Content.Builder/BuildContent.cs:309]: Seems like a neat trick.

@harry-cpp ?

Comment (harry-cpp) [reply Tools/MonoGame.Content.Builder/BuildContent.cs:309]: Yea, we are gonna have to add support to mgcb-editor for it to be able to add symbols to reference paths.

Comment (Taronyu) [reply Tools/MonoGame.Content.Builder/BuildContent.cs:309]: From the GUI you mean? As far as I understood ticket #7232 it looked like a CLI/manual call but I must admit that I'm fairly new to the project.

Comment (tomspilman) [reply Tools/MonoGame.Content.Builder/BuildContent.cs:309]: @harry-cpp can you point @Taronyu at the code for that in the editor?

Comment (harry-cpp) [reply Tools/MonoGame.Content.Builder/BuildContent.cs:309]: It does not brake anything, just adds it to the CLI, I'll fix the editor when I get the chance.

**Pull request merged**